package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghostfolio",
-  "version": "2.110.0",
+  "version": "2.111.0",
```

Comment:
Please keep at `2.110.0`, the version will be increased in the release process 🙂